test/TEST-03-JOBS/test.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 TEST_DESCRIPTION="Job-related tests"
 TEST_NO_QEMU=1
+IMAGE_NAME="default"
```

Comment:
Why this chunk?

Reply:
That's just a dumb kludge for the even dumber `test/init-integration-tests.sh` script, so it initializes the `default` image as well in one go (as the first initialization of each image has to be done before running any tests in parallel, otherwise they'll contend for the *first* initialization and die horribly).

Nevertheless, I think I'll move the script to the CentOS CI one, as it's somewhat pointless having it in this repo (and probably tweak it so it doesn't need this kludge).